{"uuid": "a6530e57-a6c5-4771-b925-dfbb714fbc8f", "children": ["3d1b30f8-cc20-4dfb-b4c8-b03f03d775f6", "8b17b445-fb29-43c6-bea9-674411ccf84c"], "befores": [{"name": "firmware_upgrade", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/conftest.py\", line 664, in firmware_upgrade\n    upgrade_status = fixtures_ver.setup_firmware(get_apnos, get_configuration)\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 155, in setup_firmware\n    current_version = str(ap_version).split(\"/\")[1].replace(\" \", \"\").splitlines()[0]\n"}, "attachments": [{"name": "firmwares/limit=10000&deviceType=cig_wf194c&latestonly=offset=3000", "source": "924100f5-8e58-432a-9ee0-52d5ca8fcd23-attachment.json", "type": "application/json"}, {"name": "Target firmware : ", "source": "1737feab-21b3-4cc0-92dd-14a3807e3877-attachment.attach"}], "start": 1638911426582, "stop": 1638911557279}], "start": 1638911426582, "stop": 1638914803185}